modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/AssignmentsTracker.java

```diff
@@ -183,7 +193,8 @@ private WatchListener createPendingAssignmentsListener() {
     private static void handleReceivedAssignments(
             WatchEvent event,
             byte[] assignmentsMetastoreKeyPrefix,
-            Map<ReplicationGroupId, TokenizedAssignments> groupIdToAssignmentsMap
+            Map<ReplicationGroupId, TokenizedAssignments> groupIdToAssignmentsMap,
+            Function<byte[], Set<Assignment>> deserFunction
```

Comment:
up to you, but I would prefer `deserializationFunction` or `deserializer`, because we dont usually use such contractions and there is no javadoc